chore: added note for vercel/netlify using serverSideTranslations (#1576)
isaachinman/next-i18next: README.md

## Notes

### Vercel and Netlify

> These services are not able to directly locate the path of your translations and requires additional configuration. If you are facing issues using `serverSideTranslations` then setting `config.localePath` to use `path.resolve` should resolve this. An example can be [found here](https://github.com/isaachinman/next-i18next/issues/1552#issuecomment-981156476).

Reviewer: ```suggestion
Some serverless PaaS may not be able to locate the path of your translations and require additional configuration. If you have filesystem issues using `serverSideTranslations`, set `config.localePath` to use `path.resolve`. An example can be [found here](https://github.com/isaachinman/next-i18next/issues/1552#issuecomment-981156476).
```
